gen/quick_test.go

```diff
@@ -144,6 +144,63 @@ func enumValueGenerator(valuesFunc interface{}) func(*testing.T, *rand.Rand) thr
 	}
 }
 
+func keyValueSetValueArgsGenerator() func(*testing.T, *rand.Rand) thriftType {
+	keyGenerator := defaultValueGenerator(reflect.TypeOf(tf.Key("")))
+	valueGenerator := unionValueGenerator(tu.ArbitraryValue{})
+	return func(t *testing.T, rand *rand.Rand) thriftType {
+		return &tf.KeyValue_SetValue_Args{
+			Key:   keyGenerator(t, rand).(*tf.Key),
+			Value: valueGenerator(t, rand).(*tu.ArbitraryValue),
+		}
+	}
+}
+
+func keyValueSetValueV2ArgsGenerator() func(*testing.T, *rand.Rand) thriftType {
+	keyGenerator := defaultValueGenerator(reflect.TypeOf(tf.Key("")))
+	valueGenerator := unionValueGenerator(tu.ArbitraryValue{})
+	return func(t *testing.T, rand *rand.Rand) thriftType {
+		return &tf.KeyValue_SetValueV2_Args{
+			Key:   *keyGenerator(t, rand).(*tf.Key),
+			Value: valueGenerator(t, rand).(*tu.ArbitraryValue),
+		}
+	}
+}
+
+func keyValueGetValueResultGenerator() func(*testing.T, *rand.Rand) thriftType {
+	successGenerator := unionValueGenerator(tu.ArbitraryValue{})
+	doesNotExistGenerator := defaultValueGenerator(reflect.TypeOf(tx.DoesNotExistException{}))
+	return func(t *testing.T, rand *rand.Rand) thriftType {
+		var result tf.KeyValue_GetValue_Result
+		if rand.Int()%2 == 0 {
+			result.Success = successGenerator(t, rand).(*tu.ArbitraryValue)
+		} else {
+			result.DoesNotExist = doesNotExistGenerator(t, rand).(*tx.DoesNotExistException)
+		}
+		return &result
+	}
+}
+
+func keyValueGetManyValuesResultGenerator() func(*testing.T, *rand.Rand) thriftType {
+	arbitraryValueGenerator := unionValueGenerator(tu.ArbitraryValue{})
+	successGenerator := func(t *testing.T, rand *rand.Rand) []*tu.ArbitraryValue {
+		values := make([]*tu.ArbitraryValue, rand.Intn(10))
```

Comment:
Not at all necessary now, but if there's ever a need we could add `numVals` as a parameter and update to:
```
func keyValueGetManyValuesResultGenerator(numVals int) func(*testing.T, *rand.Rand) thriftType {
  ...
    values := make([]*tu.ArbitraryValue, rand.Intn(numVals))
    ...
```

Reply:
We can add it in the future if we need it. There's no need for it yet.
Honestly, as long as we exercise, 0, 1, and >1, the number here doesn't
matter.